test/controllers/sections/transportUnit/TransportSealChoiceControllerSpec.scala

```diff
@@ -117,7 +117,8 @@ class TransportSealChoiceControllerSpec extends SpecBase with MockUserAnswersSer
         val result = route(application, request).value
 
         status(result) mustEqual SEE_OTHER
-        redirectLocation(result).value mustEqual routes.JourneyRecoveryController.onPageLoad().url
+        redirectLocation(result).value mustEqual
+          controllers.sections.transportUnit.routes.TransportUnitIndexController.onPageLoad(testErn, testLrn).url
```

Comment:
The test name says redirect to 'journey correction controller' I think this should be 'index controller'